library/src/scripts/headers/titleBarNavStyles.ts

```diff
@@ -146,6 +158,17 @@ const titleBarNavClasses = useThemeCache(() => {
         zIndex: 2,
     });
 
+    const navLinks = style("navLink", {
+        $nest: {
+            "&&": {
+                fontSize: unit(vars.navLinks.fontSize),
+                fontWeight: globalVars.fonts.weights.normal,
```

Comment:
Why do we take it from `globalVars` but not from  unit(vars.navLinks.fontWeight)? since we declared that in line 50?

Reply:
removed fontWeight from navLinks variable as fontWeight declared in globalVariables.

Reply:
Now I have another question about **padding** `top` and `bottom` declared but not applied. If we don't apply them, do we need them to be declared? or we probably need to apply them as well as `left` and `right`

Reply:
@NeenaDev could you answer my question please?

Reply:
i declared top, bottom into variable for future but there is no harm to remove them as we are not applying it.